lib/modules/dosomething/dosomething_zendesk/dosomething_zendesk.module

```diff
@@ -351,6 +351,20 @@ function dosomething_zendesk_verify_zendesk_user($email, $client = NULL) {
       // Create a zendesk user.
       return dosomething_zendesk_create_zendesk_user($email, $client);
     }
+    // Update user with profile fields if missing.
+    if (!$search->results[0]->user_fields->profile_url) {
```

Comment:
Hmm. What does this look like if not set, does it throw an error? Or does each user return empty value here? (Do you need to check for `isset`)